### PR TITLE
aoc/2024: day 6

### PR DESCRIPTION
**Pull request by gaverhae**

### CODE REVIEW

File: advent_of_code/2024/src/t/day06.clj [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        false (recur [y0 x0] (turn [dy dx]) cells)
+        :out (count cells)))))
+
+(defn part2
+  [{m :map, [p d] :guard}]
+  (->> (for [obs-y (range (count m))
+               obs-x (range (count (m obs-y)))
+               :when (not= [obs-y obs-x] p)]
+         [obs-y obs-x])
+       (map (fn [p] (prn p) p))
```
Comment (JustineSara) [inline advent_of_code/2024/src/t/day06.clj:47]: a little print left-over ?
```suggestion

```

**Review by JustineSara — APPROVED** [review]
I realise it's easier to read your code this year than the last, I will put it on personal progress ^^

File: advent_of_code/2024/src/t/day06.clj (hunk 1 above)
Comment (gaverhae) [reply advent_of_code/2024/src/t/day06.clj:47]: Yes; this one takes a long time to run (~25s) so I wanted to see progress. And then I disabled it so I never saw the print again.

**Comment by gaverhae** [comment]
> I realise it's easier to read your code this year than the last, I will put it on personal progress ^^

Yes, I've made quite a bit of progress in writing readable code. 😇

**Pull request merged**